mruby-sys/mruby-sys/src/mruby-sys/ext.c

```diff
@@ -54,6 +54,10 @@ struct RClass *mrb_sys_class_of_value(struct mrb_state *mrb, mrb_value value) {
   return mrb_class(mrb, value);
 }
 
+void *mrb_sys_get_ptr(mrb_value obj) {
```

Comment:
can we name this `mrb_sys_get_data_ptr`? There is a function called `mrb_get_ptr` which does a different thing 🙂